protos/dcs/common/v0/common.proto

```diff
@@ -381,3 +375,44 @@ message Vector {
   double y = 2;
   double z = 3;
 }
+
+/**
+  * Information about an object in 3D space, including its position,
+  * orientation and velocity.
+  */
+message Transform {
+  dcs.common.v0.Position position = 1;
+  // Distance between DCS' map origin to object in meters on west-east axis.
+  double u = 2;
+  // Distance between DCS' map origin to object in meters on north-south axis.
+  double v = 3;
+  Orientation orientation = 4;
+  // The direction the object is traveling to, and speed (magnitude of the
+  // vector) the object is traveling with.
+  dcs.common.v0.Vector velocity = 5;
+  // The heading of the object on a flat world. To get the heading corrected by
+  // the projection error when going from a flat to a spherical world, use
+  // `orientation.yaw` instead.
+  double heading = 6;
+  // The horizontal speed of the unit. If it is doing mach one straight up then
+  // the speed will be 0
+  double speed = 7;
+}
+
+/**
+ * The orientation of an object in 3D space.
+ */
+ message Orientation {
+  // The normalized direction the object is pointing to.
+  dcs.common.v0.Vector forward = 1;
+  // The normalized direction the three line (right wing) is pointing to.
+  dcs.common.v0.Vector right = 2;
+  // The normalized up vector (orthogonal to forward and right).
+  dcs.common.v0.Vector up = 3;
```

Comment:
Just a reminder, that `Vector` is still not a 1 to 1 mapping to DCS and rather a oppinionated modification of it (see https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/common/v0/common.proto#L373-L383)

Reply:
I do not care personally. @bitbubbler - Opinions?

Reply:
(I'd be fine with just using DCS' coordinate system and map them in my application rather than in DCS-gRPC)

Reply:
Is there a good reason to not just expose the dcs system? If one want's to port scripts from lua to something that uses grpc why force them to adopt your opinionated system?
Unless it's solving a problem.

Reply:
I reckon building a frequently calculated variables should be "opinionated." E.g. vectors, position, orientation.

However, limited to the discussion of the `Vector`, I agree with @bitbubbler as there are lots of DCS inspired Lua functions for calculating niche things, and I would be annoyed if I needed to change the vector calculations in the ported code in addition translating to another language (e.g. Lua to C#).

Let's expose those raw values (including the map's origin distance in meters) untouched, and where there is an opinion, it is extended on top of those raw values.

Reply:
I'll update the code to use the raw values, thanks for the feedback everyeone 👍